### PR TITLE
Ethos

### PR DESCRIPTION
**Pull request by leohemsted**
also a bunch of other misc cleanup. tried to keep the commits all separate and sensible.

feedback welcome i mostly just copy + pasted shit.

### CODE REVIEW

File: packages/website/src/pages/ethos.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+class EthosPage extends Component {
+  render () {
+    return (
+      <PageWrapper>
+        <Hero background={ethosBackground.src} />
+        <Panel>
+          <h2>Ethos/Code of Conduct</h2>
+
+          <p>Our club ethos and code of conduct</p>
```
Comment (leohemsted) [inline packages/website/src/pages/ethos.js]: todo write better copy than this yikes

File: packages/website/src/pages/rides.js [hunk 2]
```diff
@@ -25,11 +25,11 @@ class RidesPage extends Component {
...
           <p>First time riders are more than welcome but if you are thinking of joining us for the first time, please get in touch first so we know to look out for you!</p>
           <h3>Social Spin</h3>
-          <p>There is a club social ride on most Saturdays throughout the year and is the ideal introduction to riding with Peckham CC.</p>
+          <p>There is a club social ride on every Saturday throughout the year and is the ideal introduction to riding with Peckham CC.</p>
           <p>It's a non-drop ride, meaning we regroup at the tops of climbs and everyone rides back together.</p>
-          <p>The '<Link to='/routes'>short loop</Link>' is about 60km long and takes about three hours.  At the 30km point we usually decide whether to go further, sometimes splitting the group if people have to be back early.</p>
+          <p>The '<Link to='/routes'>short loop</Link>' is about 60km long and takes about three hours. There's always multiple groups at the library, and there's generally a group of people interested in riding further and/or slightly faster.</p>
```
Comment (achingbrain) [inline packages/website/src/pages/rides.js]: `There's` is a contraction of `There is`, so this sentence reads `There is always multiple groups` which reads a bit wonkily to me.  I should probably be `There are always multiple groups`

```suggestion
          <p>The '<Link to='/routes'>short loop</Link>' is about 60km long and takes about three hours. There are always multiple groups at the library, and there's generally a group of people interested in riding further and/or slightly faster.</p>
```

File: packages/website/src/pages/rides.js [hunk 3]
```diff
@@ -25,11 +25,11 @@ class RidesPage extends Component {
...
           <h3>Ladies Ride</h3>
-          <p>On the third Sunday of every month there is a women-only ride aimed at riders of all abilities.  The route varies month to month but is about 50km in length and the group leaves the library at 9am.</p>
+          <p>During summer, once a month there is a women-only ride aimed at riders of all abilities. The route varies month to month and is designed to offer women and non-male identifying people a safe and inclusive space to stretch themselves. The pace is inclusive and the distance is generally slightly further than the social ride.</p>
```
Comment (achingbrain) [inline packages/website/src/pages/rides.js]: I don't think this is true, summer official starts at the solstice (e.g. end of June) but the chat in the room is to start the rides up again at the beginning of March.  Maybe say something like every month from spring to autumn?

Also the description of the pace - as you say people want to stretch themselves.  It's not exclusive but from what I gather it is supposed to be somewhere between the social and a training ride.

File: packages/website/src/pages/ethos.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+import { Link } from 'react-router-dom'
+// import styled from 'styled-components'
+import { PageWrapper, Panel, Hero, LinkPanel } from '../components/panels'
+import ethosBackground from '../../assets/ethos-bg.png'
+import membershipBackground from '../../assets/membership-bg.jpg'
+import equipmentBackground from '../../assets/equipment-bg.jpg'
+
+// const SubList = styled.ol`
+//   list-style-type: lower-alpha;
+// `
```
Comment (achingbrain) [inline packages/website/src/pages/ethos.js]: Please remove the commented code.

File: packages/website/src/pages/leading.js [hunk 5]
```diff
@@ -26,6 +26,7 @@ class LeadingPage extends Component {
           <h3>Before the ride</h3>
           <p>Before setting off ensure:</p>
           <ol>
+            <li>No group is larger than ~14 people. If it is, make sure the group splits in to two.</li>
```
Comment (achingbrain) [inline packages/website/src/pages/leading.js]: Wrong place for this discussion but 14 is still a pretty big group!

Comment (leohemsted) [reply packages/website/src/pages/leading.js]: yeah you're probably right, I'll drop it down to 12. At least that way people should feel guilty if it grows to 14.

**Pull request merged**